Python/chapter01/1.1 - Is Unique/jesus_1.1.sol.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+
+def is_unique(s: str) -> bool:
+    for x in s:
+        if s.count(x) > 1:
+            return False
+    return True
+
+class TestIsUnique(unittest.TestCase):
+
+    def test_is_unique(self):
+        self.assertEqual(is_unique('aaaaa'), False)
```

Comment:
there are better assertions you can use: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue

```suggestion
        self.assertFalse(is_unique('aaaaa'))
```

same below